src/main/compose/docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '2.2'
+
+services:
+  postgresqlalpha:
+    image: ${DOCKER_IMAGE}
+    volumes:
+    - postgres-alpha_data:/var/lib/postgresql/data
+    ports:
+    - 0.0.0.0::5432
+    - 0.0.0.0::8008
+    environment:
+    - PGDATA=/var/lib/postgresql/data/PGDATA
+    - PATRONI_NAME=postgresqlalpha
+    - PATRONI_SCOPE=alfdb
+    - PATRONI_POSTGRESQL_CONNECT_ADDRESS=postgresqlalpha:5432
+    - PATRONI_RESTAPI_CONNECT_ADDRESS=postgresqlalpha:8008
+
+  postgresqlbravo:
+    image: ${DOCKER_IMAGE}
+    volumes:
+    - postgres-bravo_data:/var/lib/postgresql/data
+    ports:
+    - 0.0.0.0::5432
+    - 0.0.0.0::8008
+    environment:
+    - PGDATA=/var/lib/postgresql/data/PGDATA
+    - PATRONI_NAME=postgresqlbravo
+    - PATRONI_SCOPE=alfdb
+    - PATRONI_POSTGRESQL_CONNECT_ADDRESS=postgresqlbravo:5432
+    - PATRONI_RESTAPI_CONNECT_ADDRESS=postgresqlbravo:8008
+
+  haproxy:
```

Comment:
is this haproxy used for the build/tests?

Reply:
this was copied from docker-patroni